lib/file_codec.py

```diff
@@ -49,16 +49,7 @@ def _EncodeFile(self, parameters, bitrate, videofile, encodedfile):
         raise Exception("Encode failed with returncode %d" % returncode)
       return (subprocess_cpu, elapsed_clock)
 
-  def Execute(self, parameters, bitrate, videofile, workdir):
-    encodedfile = '%s/%s.%s' % (workdir, videofile.basename, self.extension)
-    subprocess_cpu, elapsed_clock = self._EncodeFile(parameters, bitrate,
-                                                       videofile, encodedfile)
-    result = {}
-
-    result['encode_cputime'] = subprocess_cpu
-    result['encode_clocktime'] = elapsed_clock
-    bitrate = videofile.MeasuredBitrate(os.path.getsize(encodedfile))
-
+  def _DecodeFile(self, videofile, encodedfile, workdir):
     tempyuvfile = "%s/%stempyuvfile.yuv" % (workdir, videofile.basename)
     if os.path.isfile(tempyuvfile):
       print "Removing tempfile before decode:", tempyuvfile
```

Comment:
drive-by tip: open https://docs.python.org/2/library/os.html#os.devnull instead of /dev/null.